pkgs/build-support/node/build-npm-package/hooks/npm-config-hook.sh

```diff
@@ -96,6 +107,8 @@ npmConfigHook() {
       rm node_modules/.meow
     fi
 
+    patchShebangs node_modules
```

Comment:
```suggestion
    patchShebangs node_modules/.bin
```
Would that already be enough? Searching through everyfile in the blackhole could take a bit.

Reply:
It *could* be enough, but who knows what files packages add in their install scripts that have hardcoded shebangs...

Reply:
If patchShebangs would be free, we would just run it on every source file and patch everything we can. node_modules is notorious for being very big and with lots of files.

How about we just search for executable files and patch those?

Reply:
Maybe?

```
find node_modules -executable -exec patchShebangs {} \;

Reply:
We'll get this in without addressing this for now.